transport.go

```diff
@@ -34,13 +34,11 @@ type Transporter interface {
 	// TODO: Use a better name.
 	IsConnected() bool
 
-	Run() error
-	RunAsync() <-chan error
-	RegisterProtocol(p Protocol) error
-}
+	// ReceiveFramesAsync starts processing incoming frames in a
+	// background goroutine, if it's not already happening.
+	ReceiveFramesAsync()
```

Comment:
Done. Changed `RunAsync` into `Run`, too. I'll fix gregor in a separate PR, too (since it doesn't vendor RPC).